doc/rfd/0000/README.md

```diff
@@ -0,0 +1,109 @@
+---
+authors: Jaime Gomes <jaime.gomes@suse.com>
+state: discussion
+---
+
+# Request for Discussion
+
+The goal is to have an constructive discussion within the team about an idea and capture it in a
+document known by RFD.
+
+## Scope
+
+## Metadata
+
+1. authors: owners of the RFD must be listed with their name and email address 
+(John Doe <john.doe@wonder.land>).
+2. state: one of the states discussed below.
+
+## State
+
+1. draft
+2. discussion
+3. published
+4. commited
+5. abandoned
+
+The ```draft``` state indicates that the work is not ready for discussion and that a placeholder
+was set. The RFD owner is puting his thoughs in place before promoting to ```discussion```.
+
+Documents under active discussion within the team should be in ```discussion``` state. When reaching
+this state the scope of the RFD must be clear and well defined.
+
+Once the discussion converges to an agreement then the state should be set to ```published```.
+
+Once well defined, the state should be in ```commited``` state. Any comment in the ```commited```
+state should be raised as issues or if the comment brings fundamental changes to the original
+content of the RFD then a new RFD should be created.
+
+Finally, if an idea will never be implemented or it should be ignored then it can be moved to the
+```abandoned``` state at any time.
+
+## Workflow
+
+Some of these steps may have some script support in the future to make everyones life easier.
+
+### Find Next RFD Number
+
+Check for the next RFD number by going through the RFDs table.
```

Comment:
This + next step can generate a template, maybe?

Reply:
I am trying to avoid having templates but probably we could have one just to fill the metadata.